cmd/node/main.go

```diff
@@ -604,6 +610,44 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 	return nil
 }
 
+func startStatusPolling(ash core.AppStatusHandler, pollingInterval int, networkComponents *factory.Network) error {
+	if ash != nil {
+		pollingInterval := time.Duration(pollingInterval) * time.Second
+		appStatusPollingHandler, err := appStatusPolling.NewAppStatusPolling(ash, pollingInterval)
+
+		if err != nil {
+			return errors.New("cannot init AppStatusPolling")
+		} else {
+			numOfConnectedPeersHandlerFunc := func(appStatusHandler core.AppStatusHandler) {
+				numOfConnectedPeers := int64(len(networkComponents.NetMessenger.ConnectedAddresses()))
+				appStatusHandler.SetInt64Value(core.MetricNumConnectedPeers, numOfConnectedPeers)
+			}
+
+			err = appStatusPollingHandler.RegisterPollingFunc(numOfConnectedPeersHandlerFunc)
+
+			if err != nil {
+				return errors.New("cannot register handler func for num of connected peers")
+			} else {
```

Comment:
Same as above, it would be more readable without the `else` branch

Reply:
done